README.md

```diff
@@ -769,6 +769,57 @@ wsServer.on('connection', (socket, request) => {
 
 </details>
 
+<details id="uws-auth-handling">
+<summary><a href="#uws-auth-handling">🔗</a> Server usage with <a href="https://github.com/uNetworking/uWebSockets.js">uWebSockets.js</a> and custom auth handling</summary>
```

Comment:
I think this example is redundant (since we have the exact same thing for `ws`). Besides, you cannot set custom WebSocket upgrade headers so it is always preferred to do the auth through `connectionParams`.

Reply:
> you cannot set custom WebSocket upgrade headers from the browser 
Yes, but we use cookies to auth WebSocket upgrade requests, for example.

Reply:
Sure, but there is already an exact example over at `ws`.

Reply:
Ok, you can delete this example if you want.